pytest_html/plugin.py

```diff
@@ -86,7 +88,7 @@ def data_uri(content, mime_type='text/plain', charset='utf-8'):
 
 class HTMLReport(object):
 
-    def __init__(self, logfile):
+    def __init__(self, logfile, **kwargs):
```

Comment:
Yes, avoid using `**kwargs` unless you're unable to predict the arguments being passed. In our case there's a new optional argument named `self_contained` and the value should default to `False`. If we need additional arguments later we can add them.